MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MapboxApplication.java

```diff
@@ -23,7 +23,9 @@ public void onCreate() {
       setUpCrashlytics();
     }
     Mapbox.getInstance(this, getString(R.string.access_token));
-    Mapbox.getTelemetry().setDebugLoggingEnabled(true);
+    if (Mapbox.getTelemetry() != null) {
```

Comment:
If in debug mode, can throw an exception? That would be a good test case for manual QA.

Reply:
Ok, thrown exception added. 
<img width="649" alt="Screen Shot 2019-09-03 at 9 31 46 AM" src="https://user-images.githubusercontent.com/4394910/64191589-a9417400-ce2d-11e9-987c-a53d7f1aa7d9.png">